com.openwall.John.json

```diff
@@ -40,14 +43,13 @@
             "buildsystem": "simple",
             "subdir": "src",
             "build-commands": [
-                "###### build script ######",
-                "../build.sh"
+                "./configure && make"
             ],
             "sources": [
                 {
                     "type": "archive",
                     "url": "https://github.com/magnumripper/JohnTheRipper/archive/1.9.0-Jumbo-1.zip",
-                    "sha256": "48d30775dd3324b5a915c11c2fd52c21621377f76ea007c3982edb825c7f3442"
+                    "sha256": "a99e1f892a0756a6e8af5cca8a3c6a3cc497c8225f2f191811c66b18673526a6"
```

Comment:
This is strange. I bet magnum did something after the release (otherwise I won't be able to build and publish as seen [here](https://flathub.org/builds/#/builders/39/builds/534)).

Reply:
@magnumripper, just in case you have anything to say about this (it is safe to just ignore and wait jumbo-2 release).

```
"url": "https://github.com/magnumripper/JohnTheRipper/archive/1.9.0-Jumbo-1.zip",
"sha256": "48d30775dd3324b5a915c11c2fd52c21621377f76ea007c3982edb825c7f3442"                    
"sha256": "a99e1f892a0756a6e8af5cca8a3c6a3cc497c8225f2f191811c66b18673526a6"
```

Reply:
So is this a sha256sum of the old release tarball? I have absolutely no idea how/why it could have changed.

A guess though: The actual zipping might be a fetch-time thing (just like you can ask for `index.html.gz` from a web server and it will compress `index.html` on the fly). So some zip data or metadata might have changed (eg. some version detail) while the actual payload did not.